Remotion/Web/Development.WebTesting.TestSite/Properties/launchSettings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:60500",
+      "sslPort": 60543
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "launchUrl": "index.html",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "ObjectBinding.Web.Development.WebTesting.TestSite": {
+      "commandName": "Project",
+      "dotnetRunMessages": "true",
+      "launchBrowser": true,
+      "launchUrl": "index.html",
+      "applicationUrl": "https://localhost:50543;http://localhost:50500",
```

Comment:
make sure there's separate ports here from the other test sites?

Reply:
I don't think that is necessary. Currently, we don't have a tool to host all of the .NET Core testsite and if we do we should just change the port using environment variables instead